src/sentry/static/sentry/app/components/assistant/helper.jsx

```diff
@@ -111,6 +111,7 @@ const AssistantHelper = createReactClass({
               step={currentStep}
               onFinish={closeGuide}
               onDismiss={this.handleDismiss}
+              orgSlug={GuideStore.state.currentOrgSlug}
```

Comment:
I think we're not supposed to query the store directly (@lynnagara confirm?). We should use react's message passing and set the slug as the component's state in `onGuideStateChange`.